draft-ietf-privacypass-protocol.md

```diff
@@ -287,6 +299,33 @@ authorization tokens from the server.
   store[server.id].push(tokens)
 ~~~
 
+Note that the first round of the protocol is only necessitated for
+certain ciphersuites that require client and servers commit to some
+value. When such commitment `com` is generated and sent to the client,
+the client returns `com` with the `IssuanceRequest` message. The server
+MUST check that the commitment corresponds to `com` that was previously
```

Comment:
When the commitment does not corresponds to `com` that was previously
committed, should this define a specific error that we need to add to the `Error types`?

Reply:
@claucece Good point! Yes it should be.